zombies.js

```diff
@@ -7,7 +7,14 @@
  * @param {string} name     The item's name.
  * @property {string} name
  */
-
+class Item {
+  constructor(name) {
+    this._name = name;
+  }
+  get name() {
```

Comment:
All getters should have an accompanying setter method as well. Please update all the getter with setters throughout the file.